app/messages/send-notification.js

```diff
@@ -0,0 +1,18 @@
+import { v4 as uuidv4 } from 'uuid'
+import { NotifyClient } from 'notifications-node-client'
+
+const sendNotification = async (message) => {
+  const notifyClient = new NotifyClient(process.env.NOTIFY_API_KEY)
+
+  await notifyClient.sendEmail(
+    message.body.data.notifyTemplateId,
+    message.body.data.commsAddress, {
+      personalisation: {
+        reference: message.body.data.personalisation.reference,
+        agreementSummaryLink: message.body.data.personalisation.agreementSummaryLink
+      },
+      reference: uuidv4()
```

Comment:
Fun fact.  If you just want a uuid, you can do this using default node stuff without this package.

```
import crypto from 'crypto' // default node, no need to install through npm
const uuid = crypto.randomUUID()
```

One less dependency to manage

Reply:
Done. Have also removed the `uuid` package as a dependency.